src/SocketLeakDetection.Tests/SocketLeakDetection.Tests.csproj

```diff
@@ -6,13 +6,20 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(TestSdkVersion)" />
```

Comment:
Revert this change -keep the `<PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(TestSdkVersion)" />` - we have no idea what the 16.0.1 SDK does yet. It's probably VS19.